testing/e2e/manifests/clientServer.go

```diff
@@ -78,7 +78,7 @@ func ClientAndServer(namespace, name, nameserver, keyvaultURI, host, tlsHost str
 
 	serverName := name + "-server"
 	serverDeployment := newGoDeployment(serverContents, namespace, serverName)
-	serviceName := name + "service"
+	serviceName := name + "-service"
```

Comment:
Check on this, remove if not needed

Reply:
I remember making this change because without it the names of the service wasn't separated in a similar way to the server and ingress. It's a cosmetic change only and doesn't affect how the tests run at all but I think it should stay, personally